scripts/tests/chiptest/test_definition.py

```diff
@@ -208,7 +210,7 @@ class ApplicationPaths:
     energy_management_app: typing.List[str]
 
     def items(self):
-        return [self.chip_tool, self.all_clusters_app, self.lock_app,
+        return [self.chip_tool, self.all_clusters_app, self.energy_gateway_app, self.lock_app,
```

Comment:
```suggestion
        return [self.chip_tool, self.all_clusters_app, self.lock_app,
```
Already on line 216 below